transports/tcp/Cargo.toml

```diff
@@ -10,16 +10,24 @@ keywords = ["peer-to-peer", "libp2p", "networking"]
 categories = ["network-programming", "asynchronous"]
 
 [dependencies]
-async-std = { version = "1.6.5", optional = true }
-futures = "0.3.1"
+async-io-crate = { package = "async-io", version = "1.2.0", optional = true }
+futures = "0.3.8"
 futures-timer = "3.0"
-if-addrs = "0.6.4"
+if-watch = { version = "0.1.4", optional = true }
+if-addrs = { version = "0.6.4", optional = true }
 ipnet = "2.0.0"
+libc = "0.2.80"
 libp2p-core = { version = "0.26.0", path = "../../core" }
-log = "0.4.1"
-socket2 = { version = "0.3.12" }
-tokio = { version = "0.3", default-features = false, features = ["net"], optional = true }
+log = "0.4.11"
+socket2 = { version = "0.3.17", features = ["reuseport"] }
+tokio-crate = { package = "tokio", version = "0.3", default-features = false, features = ["net"], optional = true }
```

Comment:
Maybe we can use tokio 1.0?

Reply:
I would suggest we do the upgrade in a separate pull request, see https://github.com/libp2p/rust-libp2p/pull/1907/ and https://github.com/libp2p/rust-libp2p/pull/1904.